drv/stm32h7-rng/src/main.rs

```diff
@@ -23,73 +25,131 @@ use userlib::*;
 
 task_slot!(SYS, sys);
 
+counted_ringbuf!(Trace, 32, Trace::Blank);
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Count)]
+enum Trace {
+    Blank,
```

Comment:
since this is a `counted_ringbuf` now, the `Blank` variant ought to have 
```suggestion
    #[count(skip)]
    Blank,
```